### ISSUE TITLE
Alert documentation

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#321]
The following updates are needed in the Alert component documentation:

- Update the row for `role="alert"` to something like this:

  | Attribute | Applied To | Outcome |
  | -- | -- | -- |
  | `role="alert"` | `.pf-c-alert__body` | Communicates contents of the alert message to the user. This should only be used on dynamically created alerts, such as toast notifications or inline notifications that aren't present during page load (for example, an alert at the top of a form when there are errors in the form). |

  Originally it was undecided how we wanted to address the concern of including actions in an alert, and whether these should be part of the `alert="role"` element. After thinking about this, I think that either way, the attribute should only be applied to the element that includes the message. I included additional comments about this in issue #299.

- Include classes in the Usage section that are currently missing, like  `.pf-c-alert__body`, `.pf-c-alert__title`, etc...
